content/Project-Goals-2025-March-Update.md

```diff
@@ -0,0 +1,1689 @@
++++
+layout = "post"
+date = 2025-04-04
```

Comment:
```suggestion
date = 2025-04-08
```

Reply:
It is now. 

This is a cool thing of the new blog setup: it's easier to fix without requiring to change the file name.